cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -1113,7 +1036,10 @@ impl LockedManifestCatalog {
         } else {
             manifest_systems
         };
-        Ok((valid_systems, requested_systems))
+        Ok(requested_systems
```

Comment:
suggestion (NB): readability?

```suggestion
        Ok(Vec::from_iter(&requested_systems - &valid_systems))
```